index.php

```diff
@@ -44,7 +46,11 @@
                     <?php endif; ?> 
                     
         <!-- CLOSES CALLS/QUERIES FOR MOBILE/STICKY CARDS -->
-    
+            </div>
+            </div>
+</div>
```

Comment:
I'm note entirely sure these closing tags are all needed - it looks like this diff introduces one (`.wrap-sticky`). The template is unbalanced, but the closing tags may be in the render*() functions.

I'm digging into this.